src/wp-includes/class-wp-user.php

```diff
@@ -61,7 +61,7 @@ class WP_User {
 	 * @var bool[] Array of key/value pairs where keys represent a capability name
 	 *             and boolean values represent whether the user has that capability.
 	 */
-	public $caps = array();
+	protected $caps = array();
```

Comment:
Why is this changed to `protected`? Wouldn't it be better to make it `private` instead?

Reply:
I don't think we can make this private. If someone is extending the WP_User class, then this property will then become unaccessible. There is no need for it to be private, protected will work here.

Reply:
Fair point. 👍